virtual_ecosystem/models/animal/animal_cohorts.py

```diff
@@ -931,213 +932,106 @@ def delta_mass_predation(
 
         return total_consumed_mass
 
-    def calculate_consumed_mass_herbivory(
+    def default_consumed_resource_mass(
```

Comment:
I'm not sure if there's going to be ever a consumed resource mass other than this one, but if there is not and the only point here is to re-use this same code by other methods of the class, the usual approach is to just make it "private" and remove the `default`. Otherwise, it will be understood that there is an option for not being default:

```suggestion
    def _consumed_resource_mass(
```

Reply:
Ah, good to know, thanks!